CourseGrab/Home/HomeTableViewCell.swift

```diff
@@ -8,7 +8,152 @@
 
 import Foundation
 import UIKit
+import SnapKit
 
 class HomeTableViewCell: UITableViewCell {
     
+    var section: Section? {
+        didSet {
+            guard let section = section else { return }
+            
+            courseLabel.text = String(section.courseNum)
+            enrollButton.isHidden = section.status != .open
+            sectionLabel.text = section.section
+            statusBadge.image = section.status.icon
+            titleLabel.text = section.title
+
+            if section.status == .open {
+                removeButton.snp.makeConstraints { make in
+                    make.top.equalTo(sectionLabel.snp.bottom).offset(12)
+                    make.leading.equalTo(16)
+                    make.height.equalTo(24)
+                    make.trailing.equalTo(container.snp.centerX).inset(9)
+                    make.bottom.equalTo(container).inset(16)
+                }
+            } else {
+                removeButton.snp.remakeConstraints { make in
+                    make.top.equalTo(sectionLabel.snp.bottom).offset(12)
+                    make.leading.equalTo(16)
+                    make.height.equalTo(24)
+                    make.trailing.equalTo(container).inset(16)
+                    make.bottom.equalTo(container).inset(16)
+                }
+            }
+        }
+    }
+    
+    private let container = UIView()
+    private let courseLabel = UILabel()
+    private let enrollButton = UIButton(type: .roundedRect)
+    private let removeButton = UIButton(type: .roundedRect)
+    private let sectionLabel = UILabel()
+    private let statusBadge = UIImageView()
+    private let titleLabel = UILabel()
+    
+    override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+        
+        // setup appearance
+        
+        selectionStyle = .none
+        
+        container.translatesAutoresizingMaskIntoConstraints = false
```

Comment:
when you use SnapKit you actually don't need to set `translatesAutoresizingMaskIntoConstraints` at all so you can remove all cases where you do

Reply:
I already made these changes. You are commenting on outdated code.

Reply:
oh shoot my bad